assets/js/blocks/checkout/inner-blocks/checkout-shipping-methods-block/block.tsx

```diff
@@ -54,8 +54,22 @@ const Block = (): JSX.Element | null => {
 		needsShipping,
 		isLoadingRates,
 		hasCalculatedShipping,
+		isCollectable,
 	} = useShippingData();
 
+	const filteredShippingRates = isCollectable
+		? shippingRates.map( ( shippingRatesPackage ) => {
+				return {
+					...shippingRatesPackage,
+					shipping_rates: shippingRatesPackage.shipping_rates.filter(
+						( shippingRatesPackageRate ) =>
+							shippingRatesPackageRate.method_id !==
+							'local_pickup'
```

Comment:
I will leave this feedback in the second PR, but I'd love we don't hardcode this value, but instead get it from the server while being filterable, I can see some plugins wanting to hook in and have their method receive the same treatment.

Reply:
Let's discuss this later. I think we should tie this block to a specific method.